values.yaml

```diff
@@ -6,7 +6,7 @@
 ##
 image:
   # Image pull policy. Change to "IfNotPresent" when switching to stable images
```

Comment:
```suggestion
  # Image pull policy
```
Looks like this recommendation in comments could be removed now.

Reply:
Removed, also added Changelog